quinn/src/streams.rs

```diff
@@ -149,6 +151,25 @@ where
         conn.wake();
     }
 
+    /// Monitor peer's reset event
+    ///
+    /// This will return the reason of reset when receiving
+    pub fn poll_reset(&mut self, cx: &mut Context) -> Poll<Option<VarInt>> {
```

Comment:
Resets are already exposed via `ReadError::Reset`.

Reply:
Here I've chosen to offer the same method name as `h2`. But essentially, this monitors `STOP_SENDING` events. Using QUIC's vocabulary would maybe make more sense, @djc?

Reply:
Using "reset" to describe `STOP_SENDING` is specifically incorrect, yeah.

Reply:
Is `poll_stopped()` good enough?